sunpy/util/config.py

```diff
@@ -146,3 +150,35 @@ def _get_user_configdir():
     if not _is_writable_dir(configdir):
         raise RuntimeError(f'Could not write to SUNPY_CONFIGDIR="{configdir}"')
     return configdir
+
+
+def write_config(overwrite=False):
```

Comment:
Can the name of this be changed to `copy_default_config`? This is a bit more descriptive and accurate.

Reply:
Done.